src/test/snapshot/App.test.js

```diff
@@ -3,7 +3,17 @@ import ReactDOM from 'react-dom';
 import App from '../../main/App';
 import renderer from 'react-test-renderer';
 
-test('App component', () => {
+
+let reportStatsData =
+    {
+        "minutes": 455,
+        "talks": 10,
+        "learning": "JS, Polymer, Java" ,
```

Comment:
app data for snapshot should be a complete dataset of all info in the app. Fill in all props not just report stats